Amend the previous ADR022

paroxp opened the pull request:
## What

We've chosen to go with node for PaaS Admin tool. This should be
recorded for historical references along with the reasoning behind the
decision.

## How to review

- Read through the content and make sure it's sensible

alext commented on docs/architecture_decision_records/ADR024-web-app-language-and-framework-selection-2.md:
```diff
@@ -0,0 +1,49 @@
+# Decision
+
+We will use Node to create new user-facing applications that have no need to
+interact with database.
```
I'm not sure this is quite the correct way to phrase this. It's not about connecting to a database, it's about where the application logic lives. In the [current GDS way guidance](https://gds-way.cloudapps.digital/standards/programming-languages.html#frontend-development) on this it says:

> Node.js can be used to render a web interface for your service but should not be used to implement any significant ‘application logic’.

I reckon we should phrase it similarly here.

paroxp replied:
Addressed. Thanks.

alext commented:
Looks good to me. I'll let @chrisfarms give it a once-over before it's merged.

alext commented:
This also needs a rebase now that #195 has been merged.

46bit commented:
cc @chrisfarms if you'd like to have a read before merging

paroxp commented:
ℹ️ Rebased and adapted to a new format.

chrisfarms commented on source/architecture_decision_records/ADR022-web_app_language_and_framework_selection.html.md.erb:
```diff
@@ -1,5 +1,11 @@
 
+## Superceded
+
+This has been superceded by [ADR024](/architecture_decision_records/ADR024-web-app-language-and-framework-selection-2/) since February 2018. It has been retained for historical purposes.
```
Looks like this is a dead link... possibly related to the fact the filenames have recently changed to end in `.html.md.erb` not `.erb`

paroxp commented:
It's actually due to the inconsistency in file naming. One is using `-` and one `_` to replace spaces. Joy.

Also added the link to the index file, which apparently now is not build automatically.

Fixed and TESTED.
